src/hooks/useScrollToOpenCollapse.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+
+export function useScrollToOpenCollapse() {
```

Comment:
I looks like this file is 4 space per ident instead of 2. We should run prettier on everything to ensure consistent formatting